packages/rocketchat-message-snippet/client/lib/collections.js

```diff
@@ -0,0 +1 @@
+this.Messages = new Meteor.Collection('rocketchat_message');
```

Comment:
We should not use the real collection name here, can you replace by `rocketchat_snippeted_message`